devicetypes/widomsrl/widom-smart-dry-contact.src/widom-smart-dry-contact.groovy

```diff
@@ -0,0 +1,325 @@
+/**
+ *  Widom Smart DRY contact
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *	  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "WiDom Smart Dry Contact", namespace: "WiDomsrl", author: "WiDom srl", ocfDeviceType: "oic.d.switch", mnmn: "SmartThings", vid: "generic-switch") {
+		capability "Actuator"
+		capability "Switch"
+		capability "Configuration"
+		capability "Health Check"
+
+		fingerprint mfr: "0149", prod: "1214", model: "0900", deviceJoinName: "WiDom Switch" // Raw Description WiDom Dry Contact allow to turn On/Off a load with indipendent power supply.
```

Comment:
@GiuseppeWidom raw description is information describing z-wave device, it can be found in the IDE in device details section
![image](https://user-images.githubusercontent.com/41617389/113314963-16203700-930d-11eb-9413-92345da0fe14.png)

Reply:
Thanks @PKacprowiczS
I fixed Raw Description.
I made the code in this way in order to allow the end-user to use the Z-Wave configuration parameters of this device. The Z-Wave parameters can be used to change the device behaviour not only for automations or scenes. Every device has its Z-Wave configuration parameters. I think that remove the Z-Wave configuration parameters form a Z-Wave device is a pity. This is the first one (and more simple) of other devices that I'd like to add in Smartthings.

Reply:
@PKacprowiczS
I fixed the Raw description the 1st April 2021